src/App/App.jsx

```
@@ -36,9 +36,10 @@ import { getProjectIdFromLocation } from '../library/getProjectIdFromLocation'
 import { routes } from './routes'
 
 function App({ dexieCurrentUserInstance }) {
+  console.log('app')
```

ESLint

[error] Unexpected console statement.

(no-console)